ipaserver/install/dogtaginstance.py

```diff
@@ -261,7 +261,7 @@ def configure_certmonger_renewal(self):
                 iface.add_known_ca(
                     name,
                     command,
-                    dbus.Array([], dbus.Signature('s')))
+                    dbus.Array([], dbus.Signature('s')), timeout=120)
```

Comment:
It would be nice to have a code comment here, explaining briefly why we are setting the timeout to this value.

Reply:
Done.  Thanks for the review.